tests/nix_tests/utils.py

```diff
@@ -31,7 +31,7 @@
 ADDRS = {name: account.address for name, account in ACCOUNTS.items()}
 EVMOS_ADDRESS_PREFIX = "evmos"
 DEFAULT_DENOM = "aevmos"
-WEVMOS_ADDRESS = Web3.toChecksumAddress("0xcc491f589b45d4a3c679016195b3fb87d7848210")
+WEVMOS_ADDRESS = Web3.toChecksumAddress("0x5db67696C3c088DfBf588d3dd849f44266ff0ffa")
```

Comment:
This address should be a constant. And reuse here and the jsonnet for the params as well

Reply:
This is a constant and it's easier to declare here so it can be reused in other tests in the future.

Reply:
Nevermind - I just realized you can not import into jsonnet so this is ok